_posts/2017-02-12-hello-world.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Hello, World!"
+date: 2017-02-12 15:36
+author: tommy_ip
+---
+
+__*So why call ourselves "Krypton Chicken"?*__ We (the GCI 2016 winners) were
+thinking about project ideas yesterday and one of us suggested that we need a
+"cool" name. Frosty Chicken is the first one to come, but of course, that is a
+stupid name. Our conversation somehow drifted towards our favourite element in
+the periodic table.
+
+> Geeks
+
+Elements like "Argon", "Xenon", "Krypton" came up and those sounds fairly
+interesting. We decided on Krypton as our name, but as usual, it was taken on
+Github already.
+
+Someone suggested Krypton + Chicken as a joke initially, but hey, why not? We
+have been spending a fews hours making up group names already.
```

Comment:
Some kind of epic ending is needed :)
Like:

So, here we are! Power kids. Lighting chicks. Ah, don't mind.